utest.h

```diff
@@ -726,6 +729,8 @@ UTEST_WEAK int utest_main(int argc, const char *const argv[]) {
              "  --help            Show this message and exit.\n"
              "  --filter=<filter> Filter the test cases to run (EG. MyTest*.a "
              "would run MyTestCase.a but not MyTestCase.b).\n"
+             "  --list-tests      List testnames, one per line. Output names "
+             "can be passed to --filter\n"
```

Comment:
Minor nit - but all other descriptions end with a period, can you add one after --filter?

Reply:
Fair point. done